src/queue.ts

```diff
@@ -0,0 +1,155 @@
+import BN = require('bn.js')
+import gql from 'graphql-tag'
+import { Observable } from 'rxjs'
+import { Arc, IApolloQueryOptions } from './arc'
+import { Address } from './types'
+import { realMathToNumber } from './utils'
+
+export interface IQueue {
+  id: string
+  dao: Address
+  name?: string
+}
+
+export interface IQueueState {
+  id: string
+  // scheme: IScheme
```

Comment:
why no scheme?

Reply:
because the scheme properties are now part of the QueueState...